apps/fishing-map/features/workspace/context-areas/ContextAreaLayerPanel.tsx

```diff
@@ -111,6 +118,35 @@ function LayerPanel({ dataview, onToggle }: LayerPanelProps): React.ReactElement
           {isUserLayer && <Remove dataview={dataview} />}
         </div>
       </div>
+      {layerActive && DATASOURCE_WARNING.includes(dataset?.id) && (
+        <div className={cx(styles.properties, styles.dataWarning)}>
+          <div>
+            {t(
+              `datasets:${dataset?.id}.dataWarning` as any,
```

Comment:
depending on the decision at https://github.com/GlobalFishingWatch/frontend/pull/1268/files#diff-f50769714a5978c2c452a0e4491daa04fcd316493eabf1c60a9272640a206735R1037 we'll need to keep datasets or change to the default i18n namespace